dashboard/test/widgets/task_grid_test.dart

```diff
@@ -581,6 +581,81 @@ void main() {
     expect(find.byIcon(Icons.priority_high), findsNothing);
   });
 
+  testWidgets('TaskGrid shows icon for isTestFlaky tasks with multiple attempts', (WidgetTester tester) async {
+    Task taskA1 = Task()
+      ..stageName = 'A'
+      ..builderName = '1'
+      ..name = 'A'
+      ..status = TaskBox.statusSucceeded
+      ..attempts = 1
+      ..isTestFlaky = true;
+
+    Task taskA2 = Task()
+      ..stageName = 'A'
+      ..builderName = '1'
+      ..name = 'A'
+      ..status = TaskBox.statusSucceeded
+      ..attempts = 2
+      ..isTestFlaky = true;
+
+    Task taskA3 = Task()
+      ..stageName = 'A'
+      ..builderName = '1'
+      ..name = 'A'
+      ..status = TaskBox.statusSucceeded
+      ..attempts = 3
+      ..isTestFlaky = true;
+
+    Task taskB1 = Task()
+      ..stageName = 'B'
+      ..builderName = '2'
+      ..name = 'B'
+      ..status = TaskBox.statusSucceeded
+      ..attempts = 1
+      ..isTestFlaky = false;
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: TaskGrid(
+            buildState: FakeBuildState(
+              authService: MockGoogleSignInService(),
+              cocoonService: MockCocoonService(),
+            ),
+            commitStatuses: <CommitStatus>[
+              CommitStatus()
+                ..commit = (Commit()..author = 'Cast')
+                ..tasks.addAll(
+                  <Task>[
+                    taskA1,
+                    taskA2,
+                    taskA3,
+                  ],
+                ),
+              CommitStatus()
+                ..commit = (Commit()..author = 'Cast')
+                ..tasks.addAll(
+                  <Task>[taskB1],
+                ),
+            ],
+          ),
+        ),
+      ),
+    );
+    expect(find.byIcon(Icons.priority_high), findsNWidgets(1));
+
+    // check the order of the items. The flaky should be to the left and first.
+    expect(find.byType(TaskGrid).first, findsAtLeastNWidgets(1));
+
+    LatticeScrollView latticeScrollView = tester.firstWidget(find.byType(LatticeScrollView));
+    List<List<LatticeCell>> cells = latticeScrollView.cells;
+    List<LatticeCell> myCells = cells.first;
+    expect(myCells.length, 3);
+    myCells.removeAt(0); // the first element is null.
```

Comment:
The first element is the GitHub author box :-)

Reply:
Ah okay. Good to know. Thank you.